### ISSUE TITLE
[Story] Completely update all base-singer connectors, use CDK package instead

### ISSUE THREAD

**Issue by yevhenii-ldv** [airbytehq/airbyte#7244]
Part of #6164 epic

Currently we have a lot of connectors using `base-singer` package.
In this story, we need to use the classes and methods from the `СDK` instead of using them from the `base-singer` package for following connectors:

**Note**: Please, before starting the new task - please create a separate issue for it and edit the description by adding a link.

- [x] source-tempo #7645 
- [x] source-appsflyer-singer (_We have an Appsflyer connector based NOT on Singer, but exclusively on CDK, perhaps we should just delete this connector as depricated_)
- [x] source-appstore-singer
- [x] source-kustomer-singer (resolves in [PR](https://github.com/airbytehq/airbyte/pull/8430))
